travis/main.sh

```diff
@@ -40,6 +38,14 @@ check_git_status() {
 build_autotools() {
 	v --fold="bootstrap" ./bootstrap.sh
 
+	case ${TRAVIS_BUILD_TYPE} in
+	sanitizers)
+		SANITIZE_FLAGS="-fsanitize=address -fsanitize=leak -fsanitize=alignment -fsanitize=bool -fsanitize=enum -fsanitize=float-cast-overflow -fsanitize=float-divide-by-zero -fsanitize=function -fsanitize=integer-divide-by-zero -fsanitize=null -fsanitize=object-size -fsanitize=return -fsanitize=signed-integer-overflow -fsanitize=unreachable -fsanitize=unsigned-integer-overflow"
```

Comment:
hmm, making it into a general configure flag would be a good idea.  that'd make it easier for people to run on their own system (us included).  it's not something we'd want under maintainer mode though.